Papr/Assets.xcassets/map-pin-icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "defualtMarker2.png",
```

Comment:
This filename has a typo - `defualt` → `default`

Reply:
Oh, I didn't see this, nice catch!